articles/azure/azs-how-create-vmfromsnapshot-portal.md

```diff
@@ -0,0 +1,147 @@
+---
+title: How to create a virtual machine from a disk snapshot using the UKCloud Azure Stack Hub portal
+description: Provides help for creating a virtual machine from a managed disk snapshot using the portal on UKCloud for Microsoft Azure
+services: azure-stack
+author: William Turner
+reviewer: William Turner
+lastreviewed: 05/05/2020
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Create a Virtual Machine from disk snapshot
+toc_sub3:
+toc_sub4:
+toc_title: Create a virtual machine from snapshot - Portal
+toc_fullpath: Users/How To/Create a Virtual Machine from disk snapshot/azs-how-create-vmfromsnapshot-portal.md
+toc_mdlink: azs-how-create-vmfromsnapshot-portal.md
```

Comment:
maybe some more dashes? `azs-how-create-vm-from-snapshot-portal.md` ?

Reply:
Renamed in latest commit.